react/features/face-centering/middleware.js

```diff
@@ -0,0 +1,96 @@
+import {
+    CONFERENCE_JOINED,
+    CONFERENCE_WILL_LEAVE,
+    getCurrentConference
+} from '../base/conference';
+import { JitsiConferenceEvents } from '../base/lib-jitsi-meet';
+import { MiddlewareRegistry } from '../base/redux';
+import { TRACK_UPDATED, TRACK_REMOVED } from '../base/tracks';
+
+import { UPDATE_FACE_COORDINATES } from './actionTypes';
+import {
+    loadWorker,
+    stopFaceRecognition,
+    startFaceRecognition
+} from './actions';
+import { FACE_BOX_EVENT_TYPE } from './constants';
+
+MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
+    const state = getState();
+    const { faceCoordinatesSharing } = state['features/base/config'];
+
+    if (!getCurrentConference(state)) {
+        return next(action);
+    }
+
+    if (action.type === CONFERENCE_JOINED) {
+        if (faceCoordinatesSharing?.enabled) {
+            dispatch(loadWorker());
+        }
+
+        // allow using remote face centering data when local face centering is not enabled
+        action.conference.on(
+            JitsiConferenceEvents.ENDPOINT_MESSAGE_RECEIVED,
+            (...args) => {
+                if (args && args.length >= 2) {
+                    const [ { _id }, eventData ] = args;
```

Comment:
- The first parameter is the JitsiParticipant object. All `_` properties are considered private for the class. Could you please use the `getId()` method.
- The whole thing with the args check and `_id` doesn't look very readable to me. Suggestion:
```
(participant, eventData) => {
    if (!participant || !eventData) {
        return;
    }
   
    const id = participant.getId();
    .....
```
WDYT?

Reply:
sounds good